src/storage/redis_metadata.h

```diff
@@ -336,3 +338,27 @@ class HyperLogLogMetadata : public Metadata {
 
   EncodeType encode_type = EncodeType::DENSE;
 };
+
+class TDigestMetadata : public Metadata {
+ public:
+  uint64_t compression;
```

Comment:
would 64bits too large for compression argument?

Reply:
Hi @mapleFU ,

Thanks for your advice! 😊
I have updated compression and capacity to `uint32_t`.

Best Regards,
Edward